modules/osdk-cli-reference-generate-kustomize.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-cli-reference.adoc
+
+[id="osdk-cli-reference-generate-kustomize_{context}"]
+= kustomize
+
+The `generate kustomize` subcommand contains subcommands that generate link:https://kustomize.io/[Kustomize] data for the Operator.
+
+[id="osdk-cli-reference-generate-kustomize-manifests_{context}"]
+== manifests
+
+The `generate kustomize manifests` subcommand generates or regenerates Kustomize bases and a `kustomization.yaml` file in the `config/manifests` directory, which are used to build bundle manifests by other Operator SDK commands. This command interactively asks for UI metadata by default, an important component of manifest bases, unless a base already exists or you set the `--interactive=false` flag.
+
+.`generate kustomize manifests` flags
+[options="header",cols="1,3"]
+|===
+|Flag |Description
+
+|`--apis-dir` (string)
+|Root directory for API type definitions.
+
+|`-h`, `--help`
+|Help for `generate kustomize manifests`
+
+|`--input-dir` (string)
+|Directory containing existing Kustomize files
+
+|`--interactive`
+|When set to `false`, if no Kustomize base exists, an interactive command prompt is presented to accept non-inferrable metadata.
```

Comment:
I question if "non-inferrable" is going to translate well.

Reply:
Changed to "custom".